system-parachains/bridge-hubs/bridge-hub-kusama/src/bridge_to_polkadot_config.rs

```diff
@@ -157,8 +133,17 @@ parameter_types! {
 	pub PriorityBoostPerParachainHeader: u64 = 920_224_664_224_664;
 	// see the `FEE_BOOST_PER_MESSAGE` constant to get the meaning of this value
 	pub PriorityBoostPerMessage: u64 = 182_044_444_444_444;
+	// TODO: What's the correct value?
+	pub storage BridgeDeposit: Balance = constants::currency::UNITS;
```

Comment:
for testnets I used values related to the HRMP deposits from live testnets, we could use those or add some utility adapter, which would read HRMP deposit from HostConfiguration
```suggestion
	// TODO: What's the correct value? - FAIL-CI
	pub storage BridgeDeposit: Balance = constants::currency::UNITS;
```

Reply:
I have a question: is there a specific place in `runtimes` where I can get the HRMP deposit value? Or somewhere in the testnet (on `polkadot-sdk`), maybe?

Reply:
This should be fixed by https://github.com/pandres95/runtimes/pull/7